snorkel/learning/gen_learning.py

```diff
@@ -535,42 +506,55 @@ def _compile(self, L, init_deps, init_class_prior, LF_acc_priors, is_fixed):
             weight[i]['isFixed'] = False
             weight[i]['initialValue'] = np.float64(init_deps)
 
-
         #
         # Compiles variable matrix
         #
+        # Internal representation:
```

Comment:
This cleans up the factor function definitions (now, internally, the true class labels and labeling functions match except in the case of abstain, instead of them being off-by-one).